src/redux/wishlist/wishlist.reducer.js

```diff
@@ -0,0 +1,19 @@
+import { SET_WISHLIST } from './wishlist.types';
+
+const initialState = {
+  list: []
+};
+
+const wishlistReducer = (state = initialState, { type, payload }) => {
+  switch (type) {
+  case SET_WISHLIST:
+    return {
+      ...state,
```

Comment:
There is no reason to spread `state` if the state contains only one property that you are going to override